api/src/routes/team.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * This module defines the endpoints for the "/twitter" route and exports the corresponding Router.
+ */
+
+const teamRouter = require("express").Router();
+
+const teamController = require('../controllers/team');
+
+teamRouter.patch('/twitter-handle/:team_id', teamController.storeHandle);
```

Comment:
The route naming seems a bit awkward here, imo. Do you think it's more appropriate to shuffle the order to `/:team_id/twitter-handle`? My line of thinking was that in the route: first identify the id of the team -> then define the operation that the route handles (in this case `twitter-handle`).

Reply:
The reason why I did it like this is cause it doesn't let /team/:team_id for getting the team doc route so I switched it around. If it's /team/:team_id and then /team/:team_id/twitter-handle, when I'm trying to hit the first one it thinks I'm trying to hit the second one